web-ui/src/components/reviews/periods/ReviewPeriods.jsx

```diff
@@ -457,17 +473,12 @@ const ReviewPeriods = ({ onPeriodSelected, mode }) => {
                     key={`period-lia-${id}`}
                     onClick={() => onPeriodClick(id)}
                   >
-                    <Avatar>
-                      <WorkIcon />
```

Comment:
Instead of displaying the same icon for every review period, we now display an icon that indicates the status of the review period.